app/registration/views.py

```diff
@@ -246,8 +252,8 @@ def send_confirmation_email(self, request):
                 context = {
                     "confirm_url": confirm_url.url,
                     "user_email": user.email,
-                    'project_title': 'Harvard Medical School',
-                    'project_icon_url': 'https://hms.harvard.edu/sites/all/themes/hms/logo.png'
+                    'project_title': 'Harvard Medical School Department of Biomedical Informatics',
+                    'project_icon_url': 'https://portal.aws.dbmi-dev.hms.harvard.edu/static/hms_dbmi_logo.png'
```

Comment:
Can you make this a dynamic URL?

Reply:
Yeah this one was a little a tricky. It is/was surprisingly hard to get the full URL to a static file within a view (through a template it's easy). Some of the built in functions didn't account for the proxy we have with local development, where the docker thinks the URL is http://scireg:port, but instead its http://localhost, so the image wouldn't render in the email in local development. The full URL is needed because this image link gets placed within an HTML snippet that the email client renders. Any suggestions?

Reply:
Ah right it was the DEV link. I'll update it.

Reply:
All set